### PR TITLE
fix typo in changeset

### PR DESCRIPTION
**Pull request by liruifengv**
## Changes

- fix typo in changeset

### CODE REVIEW

File: .changeset/dull-carpets-breathe.md [hunk 1]
```diff
@@ -21,10 +21,10 @@ import ReactComponent from "../components/button.jsx"
 import VueComponent from "../components/button.vue"
 
 // MDX runtime is contained inside the Astro
```
Comment (florian-lefebvre) [inline .changeset/dull-carpets-breathe.md:23]: @ematipico this sentence feels incomplete, inside the Astro what?

**Review by ematipico — APPROVED** [review]

**Review by florian-lefebvre — APPROVED** [review]
LGTM! I left something for Ema

File: .changeset/dull-carpets-breathe.md (hunk 1 above)
Comment (ematipico) [reply .changeset/dull-carpets-breathe.md:23]: It should be 
> // MDX runtime is contained inside Astro

File: .changeset/dull-carpets-breathe.md [hunk 2]
```diff
@@ -21,10 +21,10 @@ import ReactComponent from "../components/button.jsx"
...
-import mdxRenderer from "@astrojs/jsx/serverr.js"
+import mdxRenderer from "@astrojs/jsx/server.js"
```
Comment (liruifengv) [inline .changeset/dull-carpets-breathe.md:24]: Oh. This should be `astro/jsx/server.js`? We don't have a `@astrojs/jsx` package?

Comment (florian-lefebvre) [reply .changeset/dull-carpets-breathe.md:24]: I think it may be an internal package so people usually dont have to install it?

Comment (syhily) [reply .changeset/dull-carpets-breathe.md:24]: @florian-lefebvre Nope, this is a misspelled package name. It's `astro/jsx/server.js` indeed.

Comment (ematipico) [reply .changeset/dull-carpets-breathe.md:24]: Another typo of mine, sorry. Probably a bug. It should be `astro/jsx/server.js`

File: .changeset/dull-carpets-breathe.md (hunk 1 above)
Comment (liruifengv) [reply .changeset/dull-carpets-breathe.md]: > // MDX runtime is contained inside the Astro core

I use it.

**Review by ematipico — APPROVED** [review]

**Pull request merged**